css/src/components/table.css

```diff
@@ -0,0 +1,168 @@
+.yoast table {
+	font-size: var(--font-size-default);
+	color: var(--color-default);
+	line-height: 1.2;
+	width: 100%;
+	border-bottom: var(--border-default);
+	border-spacing: 0;
+}
+
+.yoast tfoot th {
+	border-bottom: none;
+}
+
+.yoast th {
+	border-bottom: var(--border-default);
+}
+
+.yoast th, .yoast th.sortable {
+	color: var(--color-dark);
+	text-align: left;
+	padding: 16px;
+}
+
+.yoast th.sortable a, .yoast th a {
+	color: var(--color-dark);
+	padding: 0;
+	font-weight: 500;
+}
+
+.yoast tr {
+	height: 50px;
+}
+
+.yoast td {
+	padding: 16px;
+	border-bottom: var(--border-default);
+	font-size: var(--font-size-default);
+}
+
+.yoast .tablenav {
+	height: auto;
+	margin: 6px 0 16px;
+}
+
+.yoast .tablenav.top .tablenav-pages {
+	display: none;
+}
+
+.yoast .tablenav.top #posts-filter .tablenav-pages {
+	display: block;
+}
+
+.yoast .tablenav .yoast-button {
+	color: var(--color-dark);
+}
+
+.yoast .tablenav .search-box{
+	float: right;
+}
+
+.yoast .tablenav .search-box div {
+	float: left;
+	width: 200px;
+	margin-right: 8px;
+}
+
+.yoast .tablenav .displaying-num {
+	margin-right: 22px;
+}
+
+.yoast .tablenav .paging-input {
+	margin-right: 6px;
+}
+
+.yoast .column-actions {
```

Comment:
I can't find an element with class `column-actions`  in the bulk editor table. There's one in the Redirects table but then this should go on the CSS for Premium?